docs/contributing/code-of-conduct.md

```diff
@@ -1,50 +1,68 @@
 ---
-title: Gatsby Contributor Covenant Code of Conduct
+title: গ্যাটসবি কনট্রিবিউটরদের আচরনবিধি
 ---
 
-## Our Pledge
+## আমাদের অঙ্গীকার
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+গোষ্ঠীর অবাধ এবং স্বাগতপূর্ণ ভাবমূর্তি বজায় রাখার আগ্রহে,
```

Comment:
The whole translation should be in line no. 7 not 7-12.

Reply:
Done. Please review.